intelmq/bots/collectors/twitter/collector_twitter.py

```diff
@@ -0,0 +1,119 @@
+"""
+Twitter Collector Bot
+
+Extracts tweet_text from tweets and follows urls in supported_sources. 
+Url text and tweet text are sent separately.
+
+Parameters:
+
+    feed - feed name (Twitter)
+
+    tweet_count : int     
+
+    target_timelines : comma separated names of users whose timelines are followed
+
+    follow_urls : comma separated screen_names of tweeters for which it is allowed to follow urls, currently extraction only from pastebin is supported 
+
+    include_rts : bool default False (include retweets)
+
+    exclude_replies: bool default True
+
+    timelimit: int default 84400s, how far to the past do we accept tweets?
+
+    Api login data:
+        consumer_key
+        consumer_secret
+        access_token_key
+        access_token_secret
+
+To get api login data see: https://python-twitter.readthedocs.io/en/latest/getting_started.html
+
+"""
+
+
+import time
+import twitter
+import requests
+from intelmq.lib.bot import CollectorBot
+
+
+class TwitterCollectorBot(CollectorBot):
+
+    def init(self):
+        self.current_time_in_seconds = int(time.time())
+        self.target_timelines = []
+        if getattr(self.parameters, "target_timelines", '') is not '':
+            self.target_timelines.extend(
+                        self.parameters.target_timelines.split(','))
+        self.follow_urls = []
+        if getattr(self.parameters, "follow_urls", '') is not '':
+            self.follow_urls.extend(
+                        self.parameters.follow_urls.split(','))
+        self.include_rts = getattr(self.parameters, "include_rts", False)
+        self.exclude_replies = getattr(self.parameters, "exclude_replies", False)
+        self.timelimit = getattr(self.parameters, "timelimit", 24 * 60 * 60)
+        self.api = twitter.Api(
+                            consumer_key=self.parameters.consumer_key,
+                            consumer_secret=self.parameters.consumer_secret,
+                            access_token_key=self.parameters.access_token_key,
+                            access_token_secret=self.parameters.access_token_secret,
+                            tweet_mode="extended")
+
+    def get_text_from_url(self, url: str) -> str:
+        supported_sources = ["pastebin.com"]
+        for source in supported_sources:
+            if source in url:
+                if "raw" not in url:
+                    request = requests.get(
+                            url.replace("pastebin.com", "pastebin.com/raw"))
```

Comment:
In response to above. I remove that because what we care about is only the domain. However, I appreciate that it doesn't make sense to unnecessarily get rid of the additional information in the url. 
Ad foo.bar, I will change / to // in the regex.